app/scripts/controllers/MakeRoomCtrl.js

```diff
@@ -6,15 +6,17 @@
 	  		Room.rooms.$add({title: this.name});
 	  	}
 
-	  	this.open = function (size, parentSelector, $uibModal) {
```

Comment:
Passing $uibModal was not needed and the reason why is was undefined. $uibModal is already available and the argument broke everything.